game-core/src/main/java/games/strategy/engine/data/AttachmentProperty.java

```diff
@@ -0,0 +1,84 @@
+package games.strategy.engine.data;

```

Comment:
Note that this file seems to use CRLF for line delimiters.

Reply:
Good catch, I forgot to set my project settings on game-core, to use UTF-8 and LF